awx_collection/plugins/module_utils/tower_api.py

```diff
@@ -4,11 +4,9 @@
 from . tower_module import TowerModule
 from ansible.module_utils.urls import Request, SSLValidationError, ConnectionError
 from ansible.module_utils.six import PY2
-from ansible.module_utils.six.moves.urllib.parse import urlencode
 from ansible.module_utils.six.moves.urllib.error import HTTPError
 from ansible.module_utils.six.moves.http_cookiejar import CookieJar
 import time
-import re
```

Comment:
@john-westcott-iv it looks like these are `flake8` failures that, for whatever reason, were not getting picked up by the test targets we're running. The unused imports are largely leftover from splitting into 2 different classes.